js/detectBrowserVersion.js

```diff
@@ -0,0 +1,22 @@
+/* Try to detect FF version */
+function detectBrowser() {
+	var ECMAScriptStatus = "!",
+	    ecmaVersion = 6,
+	    userAgent = navigator.userAgent;
+
+if (navigator.userAgent.search(/Firefox/) > -1) {
```

Comment:
as this only works for Firefox, I'm really hesitant to accept this PR. See the other comment for my suggestion.